### PR TITLE
Homework9: Ekaterina Zavadskaya

### PR DESCRIPTION
**Pull request by KateZavadskaya**
Add done homework9

Docs: hm9_1.py - hm9_3.py

### CODE REVIEW

File: hm9_1.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        if a[0] == "#":
+            del a[0]
+            new_line = "".join(a)
+        elif new_line.index("#") >= 0:
+            del a[a.index("#") - 1:a.index("#") + 1]
+            new_line = "".join(a)
+    return "".join(a)
+
+
+print(line_with_symbol("abc#d##c78#"))
```
Comment (Stml89) [inline hm9_1.py]: remove it or use assert

File: hm9_1.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+""" Task 1: clean # """
+
+
+def line_with_symbol(line):
+
+    """ Function cleans symbols '#' and the symbol before """
+
+    new_line = line[:]
```
Comment (Stml89) [inline hm9_1.py]: You do not need to cope a list, work with `line`

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hm9_1.py (hunk 1 above)
Comment (KateZavadskaya) [reply hm9_1.py]: Removed "print" in all hm9.

File: hm9_1.py (hunk 2 above)
Comment (KateZavadskaya) [reply hm9_1.py]: See you, fixed 👍

**Review by Stml89 — APPROVED** [review]

**Pull request merged**